Updates example section in the Readme (#48)
Pierre-Sassoulas/pySankey: README.md

Sankey diagrams</a> flowing only from left to right.

[![PyPI version](https://badge.fury.io/py/pySankeyBeta.svg)](https://badge.fury.io/py/pySankeyBeta)
[![Build Status](https://github.com/Pierre-Sassoulas/pySankey/actions/workflows/ci.yaml/badge.svg)](https://github.com/Pierre-Sassoulas/pySankey/actions/workflows/ci.yaml)
[![Coverage Status](https://coveralls.io/repos/github/Pierre-Sassoulas/pySankey/badge.svg?branch=master)](https://coveralls.io/github/Pierre-Sassoulas/pySankey?branch=master)
[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/ambv/black)
[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)

## Examples

### Simple expected/predicted example with fruits.txt:

> `pysankey` contains a simple expected/predicted dataset called `fruits.txt` which looks
> the following:

Reviewer: ```suggestion
`pysankey` contains a simple expected/predicted dataset called `fruits.txt` which looks
like the following:
```

    'kiwi':'#9BD937'
}

labels = list(colorDict.keys())
leftLabels = [label for label in labels if label in df['true'].values]
rightLabels = [label for label in labels if label in df['predicted'].values]

# Create the sankey diagram
ax = sankey(
    left=df['true'],
    right=df['predicted'],
    leftLabels=leftLabels,
    rightLabels=rightLabels,
    colorDict=colorDict,
    aspect=20,
    fontsize=12
)

plt.show() # to display
```

![Fruity Alchemy](.github/img/fruits.png)

### Plotting preprocessed data using weights

> However, not always you have or can have the data available in the format mentioned in
> the previous example (e.g. if the dataset is too large). In this case, the weights
> between the true and predicted labels can also be calculated beforehand and used to
> create the sankey diagram. In this example, we continue to work with the data loaded
> already in the previous example:

Reviewer: ```suggestion
However, the data may not always be available in the format mentioned in
the previous example (for instance, if the dataset is too large). In such cases, the weights
between the true and predicted labels can be calculated in advance and used to
create the Sankey diagram. In this example, we will continue working with the data that was loaded
in the previous example:
```
